src/pydata_sphinx_theme/theme/pydata_sphinx_theme/sections/announcement.html

```diff
@@ -1,50 +1,18 @@
-{% set banner_label = _("Announcement") %}
-{% set header_classes = ["bd-header-announcement", "container-fluid", "init"] %}
```

Comment:
I removed the Bootstrap `container-fluid` class because the styles it provides are all now provided or overridden by the styles set directly on the banners in `_announcement.scss`.